.cicd/.helpers

```diff
@@ -42,8 +39,9 @@ function docker_tag_exists() {
     ( [[ $EXISTS =~ '404 page not found' ]] || [[ $EXISTS =~ 'manifest unknown' ]] ) && return 1 || return 0
 }
 
-if [[ $(uname) != 'Darwin' ]]; then # Darwin doesn't need hash (yet)
-    determine-hash ".cicd/${IMAGE_TAG}.dockerfile"
+# If you're on mac and you want to generate the image, use "force" as $2
```

Comment:
What does this mean?
Running the following code on your branch...
```
.cicd$ grep -r '$2' .
Binary file ./metrics/test-metrics.tar.gz matches
./.helpers:# If you're on mac and you want to generate the image, use "force" as $2
```
...shows that $2 is not consumed anywhere. If it is, we should automate this using:
```
[[ $(uname) != 'Darwin' ]]
```
But we are not building base images on mac, so I don't think we need any of this.